1회차/이상욱/3_Flatten.py

```diff
@@ -1,3 +1,15 @@
 import sys
 
 sys.stdin = open("_Flatten.txt")
+T = 10
+for t in range(1, T+1):
+    n = int(input())
+    numbers = list(map(int, input().split()))
+
+    for i in range(n):
+        numbers[numbers.index(max(numbers))] = numbers[numbers.index(max(numbers))] - 1
+        numbers[numbers.index(min(numbers))] = numbers[numbers.index(min(numbers))] + 1
```

Comment:
깔끔한 코드네요! 👍 
식에서 같은 인덱스의 리스트 원소에 접근할 때, 연산자와 등호를 합쳐서 사용할 수 있어요!
```suggestion
        numbers[numbers.index(max(numbers))] -= 1
        numbers[numbers.index(min(numbers))] += 1
```